src/pages/Explore.js

```diff
@@ -7,24 +7,43 @@ import { Bar } from 'react-chartjs-2';
 function Explore() {
   const [gunsOwnedData, setGunsOwnedData] = useState([]);
   const [gunsOwnedYears, setGunsOwnedYears] = useState([]);
```

Comment:
This all looks great, Matt! You have a lot of margin on the sides, and its a pretty large page. I would recommend you decrease the margins so the text isn't so large compared to the chart.